docs/oauth2-examples-google.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Use Google as OAuth 2.0 server
+displayed_sidebar: docsSidebar
+---
+<!--
+Copyright (c) 2005-2024 Broadcom. All Rights Reserved. The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+
```

Comment:
If this is a new topic, then I don't think we need "2005" in the copyright, see above, just have "2024" this is a GLOBAL comment for any other new topics - please confirm with JSP before changing - thanks